### PR TITLE
 Fix crash when auto instrumenting AVAsset download task

### PR DESCRIPTION
**Pull request by nachoBonafonte**
Skip tasks that are of kind of AVAssetDownloadTask. 
Most of the changes are Swift formatting changes, real fix starts in line 575

### CODE REVIEW

File: Sources/Instrumentation/URLSession/URLSessionInstrumentation.swift [hunk 1]
```diff
@@ -573,7 +571,13 @@ public class URLSessionInstrumentation {
     }
 
     private func urlSessionTaskWillResume(_ task: URLSessionTask) {
-        let taskId = self.idKeyForTask(task)
+        // AV Asset Tasks cannot be auto instrumented, they dont include request attributes, skip them
+        if let avAssetDownloadTask = NSClassFromString("__NSCFBackgroundAVAssetDownloadTask"),
```
Comment (bryce-b) [inline Sources/Instrumentation/URLSession/URLSessionInstrumentation.swift:575]: should we move this `classFromString` call  to a static const? maybe it's not very expensive, but will this be called every time this method is called? Is it necessary?

Comment (nachoBonafonte) [reply Sources/Instrumentation/URLSession/URLSessionInstrumentation.swift]: Done!!

**Comment by vvydier** [comment]
looks good as discussed on the call!

**Review by vvydier — APPROVED** [review]
looks good as discussed on the call

**Review by bryce-b — APPROVED** [review]

**Comment by nachoBonafonte** [comment]
Fixes #447

**Pull request merged**